oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -369,10 +370,12 @@ private APIGatewayProxyResponseEvent redirect(
                                             authenticationRequest.getScope()))
                             .claim(
                                     "redirect_uri",
-                                    configurationService.getOrchestrationRedirectUri())
-                            .build();
+                                    configurationService.getOrchestrationRedirectUri());
 
-            var encryptedJWT = authorizationService.getSignedAndEncryptedJWT(claims);
+            var claimsSetRequest =
```

Comment:
Are there tests that we can add for these changes?

Reply:
I don't think there are tests for the Orch/Auth request when the feature flag is enabled. 

I can look to fill the test gaps as a separate PR